codegen/src/Azure.Iot.Operations.ProtocolCompiler/T4/serialization/Object/t4/ResultAvroSchema.tt

```diff
@@ -8,7 +8,7 @@
 <# foreach (var nameDescSchemaRequiredIndex in this.nameDescSchemaRequiredIndices) { #>
     {
       "name": "<#=nameDescSchemaRequiredIndex.Item1#>",
-<#=AvroSchemaSupport.GetTypeAndAddenda(nameDescSchemaRequiredIndex.Item3, 4, this.sharedPrefix, nullable: !nameDescSchemaRequiredIndex.Item4, this.definedIds)#>
+      "type": <#=AvroSchemaSupport.GetTypeAndAddenda(nameDescSchemaRequiredIndex.Item3, 6, this.sharedPrefix, nullable: !nameDescSchemaRequiredIndex.Item4, this.definedIds)#>
```

Comment:
This is a fix for an unrelated bug in Rust generation of AVRO schema that I'm slipping into this PR.